src/genotyper.cc

```diff
@@ -664,6 +977,19 @@ Status genotype_site(const genotyper_config& cfg, MetadataCache& cache, BCFData&
         update_orig_calls_for_loss(cfg, records, bcf_nsamples, dataset_header.get(),
                                    sample_mapping, loss_trackers);
 
+        // Update format fields
+        for (auto& record : records) {
+            vector<int> allele_mapping(record->n_allele, -1);
+            vector<bool> deletion_allele(record->n_allele, false); // which alleles are deletions
+
+            S(find_allele_mapping(site, record.get(), allele_mapping, deletion_allele))
```

Comment:
I wish we could avoid computing the allele mapping twice...could it be passed into `translate_genotypes` instead of recomputed there?